sfs/mono/source.py

```diff
@@ -91,7 +145,7 @@ def point_modal(omega, x0, n0, grid, L, N=None, deltan=0, c=None):
     n0 : (3,) array_like
         Normal vector (direction) of source (only required for
         compatibility).
-    grid : triple of numpy.ndarray
+    grid : triple of array_like
```

Comment:
While you're at it, you could change the same thing for `point_modal_velocity()`